src/App.js

```diff
@@ -1,19 +1,44 @@
-import React from 'react';
-import './App.css';
+import React, { Component } from 'react';
+import TodoList from './Components/TodoList';
+import Select from './Components/Select';
+import Users from './api/users';
+import Todos from './api/todos';
 
-import users from './api/users';
+export default class App extends Component {
+  state = {
+    users: Users,
+    todos: Todos,
+  };
 
-function App() {
-  return (
-    <div className="App">
-      <h1>Static list of todos</h1>
+  updateMainState = (id, todosText) => {
+    this.setState((state) => {
+      const newTodo = {
+        userId: id,
+        id: this.state.todos.length + 1,
```

Comment:
Зачем обращаться через this, если у тебя уже есть объек со стейтом? 
```suggestion
        id: state.todos.length + 1,
```

Reply:
+